packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -102,6 +102,8 @@ export interface TaggedChange<TChangeset> {
 	 */
 	readonly rollbackOf?: RevisionTag;
 	readonly change: TChangeset;
+
+	readonly telemetryProperties?: { [key: string]: number };
```

Comment:
This type is not specifically associated with rebasing (or telemetry) so putting rebasing telemetry in it seems questionable. I would rather we change the return type of `rebaseChange` to an object that contains a `TaggedChange`.